### ISSUE TITLE
Is it possible to use the header on PCB directly?

### ISSUE THREAD

**Issue by tengg** [jonhiggs/ml67#2]
Instead jumping wires from PCB?

**Comment by jonhiggs** [comment]
Yeah, it's possible. I've described what each of the pins do somewhere. The only reason that I didn't go that way was because I couldn't find a connector to attach to the board. I think it's a 2x14 male header with a 2.00mm pitch.

If you find one, can you please post a link back?

**Comment by jonhiggs** [comment]
The pins for the header are like so:

| 28/27 | 26/25 | 24/23 | 22/21 | 20/19 | 18/17 | 16/15 | 14/13 | 12/11 | 10/09 | 08/07 | 06/05 | 04/03 | 02/01 |
|-------|-------|-------|-------|-------|-------|-------|-------|-------|-------|-------|-------|-------|-------|
| GND   | C12   | C11   | C10   | C9    | C8    | C7    | C6    | C5    | C4    | C3    | C2    | C1    | C0    |
| OC13  | LED1  | LED2  | +5v   | C15   | C14   | R7    | R6    | R5    | R4    | R3    | R2    | R1    | R0    |
